dist/browser.js

```diff
@@ -15649,11 +15665,12 @@ var ParameterDef = /*#__PURE__*/function (_Expression) {
     value: function exec(ctx) {
       // If context parameters contains the name, return value.
       if (ctx && ctx.parameters[this.name] !== undefined) {
-        return ctx.parameters[this.name]; // If default type exists, execute the default type
+        return ctx.parameters[this.name]; // If the parent context contains the name, return that
+      } else if (ctx.getParentParameter(this.name) !== undefined) {
+        var parentParam = ctx.getParentParameter(this.name);
+        return parentParam.default != null ? parentParam.default.execute(ctx) : parentParam; // If default type exists, execute the default type
       } else if (this.default != null) {
-        return this.default != null ? this.default.execute(ctx) : undefined; // Else, if context and context's parent exist return the value of the parent's parameters with the given name.
-      } else {
-        return ctx.getParentParameter(this.name);
+        this.default.execute(ctx);
```

Comment:
doesn't looks like execution was browserified. is that expected?

Reply:
It appears similar to the current contents, but perhaps it never worked correctly. 

@AndrewBird81 Does this look like accurate browserified content?